h2o-py/tests/testdir_algos/coxph/pyunit_coxph_validations.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.coxph import H2OCoxProportionalHazardsEstimator
+from h2o.exceptions import H2OResponseError
+
+
+def coxph_validations():
+    heart = h2o.import_file(pyunit_utils.locate("smalldata/coxph_test/heart.csv"))
+    heart["age"] = heart["age"].ascharacter()
+    # validate start/stop columns
+    try:
+        coxph = H2OCoxProportionalHazardsEstimator(
+            start_column="XXX",
+            stop_column="YYY"
+        )
+        coxph.train(x=["surgery", "transplant", "year"], y="event", training_frame=heart)
+        assert False
```

Comment:
Please, add error messages to all asserts in this test.

Reply:
done

Reply:
I can't see an error message for `assert False`. :) However, If you think it is not important, I am ok with these changes.

Reply:
forgot about those! added.

Reply:
Perfect! Thank you @honzasterba!